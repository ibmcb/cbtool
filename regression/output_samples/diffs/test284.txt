 status: Waiting for vm_60 (cloud-assigned uuid 1A8F4D73-A09F-5F34-A99C-19399EC01746) to start...
 status: Waiting for vm_58 (cloud-assigned uuid D6ADF1D4-D540-5D86-BD7D-B233CE2A521D) to start...
 status: Waiting for vm_59 (cloud-assigned uuid 1C4DE997-A342-544D-9F14-065A1140C0FA) to start...
 status: Trying to establish network connectivity to vm_60 (cloud-assigned uuid 1A8F4D73-A09F-5F34-A99C-19399EC01746), on IP address 79.35.254.33...
 status: Trying to establish network connectivity to vm_58 (cloud-assigned uuid D6ADF1D4-D540-5D86-BD7D-B233CE2A521D), on IP address 92.155.71.180...
 status: Trying to establish network connectivity to vm_59 (cloud-assigned uuid 1C4DE997-A342-544D-9F14-065A1140C0FA), on IP address 129.39.11.231...
 status: Bypassing the sending of a copy of the code tree to vm_58 (92.155.71.180)...
 status: Bypassing the sending of a copy of the code tree to vm_60 (79.35.254.33)...
 status: Bypassing the sending of a copy of the code tree to vm_59 (129.39.11.231)...
AI object 4DA18983-5555-508D-9A68-DC3995B55B09 (named "ai_11") sucessfully attached to this experiment. It is ssh-accessible at the IP address 92.155.71.180 (D6ADF1D4-D540-5D86-BD7D-B233CE2A521D.simcloud.com).
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |83.212.20.114   |1374526402     |none                                    |3E0E082E-3CC9-58D2-96AF-2F6559324443
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |218.3.95.25     |1374526442     |none                                    |4FD4E473-B901-535E-81B0-B1D8E6D698DB
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |86.43.50.187    |1374526461     |none                                    |F8B6C5F5-69E7-54E3-ABA6-76C0E27513C8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |40.164.186.82   |1374526548     |none                                    |65F2A248-D223-51E0-9BEC-27A2FC1F0ADF
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |207.8.217.196   |1374526554     |none                                    |8CA2C11D-7925-5F77-BEC6-B769F7254F4A
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |81.175.206.221  |1374526560     |none                                    |8C6A5E67-86B3-5B4B-8CC0-FAB15590DC8B
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |92.155.71.180   |1374526566     |none                                    |4DA18983-5555-508D-9A68-DC3995B55B09
|vm_33         |client_daytrader    |nano32      |83.212.20.114   |simhostd1                  |LG                  |ai_3    |none       |6FB95A0B-DC69-5BCA-83D4-EF6A5FDE7418
|vm_34         |was                 |iron32      |141.110.188.78  |simhostb2                  |SUT                 |ai_3    |none       |E679191D-E766-5120-B1AC-B359C5F48C44
|vm_35         |db2                 |copper32    |111.33.159.57   |simhostb4                  |SUT                 |ai_3    |none       |508E8790-2816-5B0E-B5EA-25A85B4E3E5B
|vm_36         |client_daytrader    |nano32      |218.3.95.25     |simhostd4                  |LG                  |ai_4    |none       |8262E811-541A-5386-8790-420D769BCF89
|vm_37         |was                 |iron32      |167.6.69.81     |simhostb1                  |SUT                 |ai_4    |none       |4DEF67D9-E330-5E87-8DA1-E1A5C8060BF7
|vm_38         |db2                 |copper32    |87.201.50.211   |simhostb0                  |SUT                 |ai_4    |none       |57F691F0-F925-5665-BF65-1D8523FE67EC
|vm_39         |netclient           |platinum64  |86.43.50.187    |simhosta0                  |SUT                 |ai_5    |none       |C3B97918-FB0A-5D60-B179-C521859C2641
|vm_40         |netserver           |iron32      |14.75.181.209   |simhostc2                  |SUT                 |ai_5    |none       |4CEAF1BD-1B5A-541D-B8DB-84C44699AADE
|vm_48         |lb                  |iron32      |65.48.235.103   |simhostb3                  |SUT                 |ai_8    |none       |D2149426-231B-5104-A868-4088453D2CE8
|vm_47         |was                 |iron32      |16.243.58.9     |simhostc4                  |SUT                 |ai_8    |none       |B9BDE08B-E66D-56A3-8C96-E961EDCF150B
|vm_49         |client_daytrader    |nano32      |40.164.186.82   |simhostd0                  |LG                  |ai_8    |none       |4AC7923E-F7FF-58BC-B716-E3E49B9B9AFC
|vm_50         |was                 |iron32      |222.18.150.104  |simhosta3                  |SUT                 |ai_8    |none       |8ADFD3A8-2392-5ED7-88C7-3AD3E30CC079
|vm_51         |db2                 |gold32      |125.211.95.133  |simhostc4                  |SUT                 |ai_8    |none       |E090FBC3-56D8-5DB3-B4BB-B61B6E2FE270
|vm_52         |client_daytrader    |nano32      |207.8.217.196   |simhostd0                  |LG                  |ai_9    |none       |7B86C78E-A1A7-5317-A634-98BB5A9D84D4
|vm_53         |was                 |iron32      |223.39.210.221  |simhostb0                  |SUT                 |ai_9    |none       |B6CB93B7-1C28-58F9-B9E6-31662BBF9083
|vm_54         |db2                 |copper32    |159.90.112.187  |simhostb1                  |SUT                 |ai_9    |none       |695E187B-829F-5425-B2F1-D68302DC4C4F
|vm_55         |client_daytrader    |nano32      |81.175.206.221  |simhostd1                  |LG                  |ai_10   |none       |1E77B6CB-09DA-59F0-ABEC-4973A790D91E
|vm_56         |db2                 |copper32    |77.44.214.248   |simhostb3                  |SUT                 |ai_10   |none       |AE094B27-927E-5C57-9651-45E4E70E0EAC
|vm_57         |was                 |iron32      |111.184.121.1   |simhostc2                  |SUT                 |ai_10   |none       |1121B6B8-15DC-57C1-B521-AAF61E498FD0
|vm_58         |client_daytrader    |nano32      |92.155.71.180   |simhostd2                  |LG                  |ai_11   |none       |48777964-6D45-597C-A667-BE6228D7FB07
|vm_59         |was                 |iron32      |129.39.11.231   |simhostc3                  |SUT                 |ai_11   |none       |0F709DC4-4044-522C-8C4C-C5530800B6FF
|vm_60         |db2                 |copper32    |79.35.254.33    |simhostb3                  |SUT                 |ai_11   |none       |79F46801-2AEB-5031-99D0-81CD10F49106
|Used Memory                                          |41.30M                        
|Uptime (in seconds)                                  |13565346                      
|Total Connections Received                           |6494908                       
|Total Commands Processed                             |93839694                      
|Storage Size                                         |16112607216.0                 
|Data Size                                            |13823120192.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.2543816                  
|Collections                                          |60                            
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|9                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|6                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|5                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|2                             
