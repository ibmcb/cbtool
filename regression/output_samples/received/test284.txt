 status: Waiting for vm_58 (cloud-assigned uuid 789D1360-DF3E-5826-9A36-45B7EB6E1C36) to start...
 status: Waiting for vm_59 (cloud-assigned uuid 5C3389DD-2929-5499-BE9E-E1D50F5ED9B7) to start...
 status: Waiting for vm_60 (cloud-assigned uuid D145C17D-B820-5290-ADD6-61D4C8316DB7) to start...
 status: Trying to establish network connectivity to vm_60 (cloud-assigned uuid D145C17D-B820-5290-ADD6-61D4C8316DB7), on IP address 198.69.239.253...
 status: Trying to establish network connectivity to vm_58 (cloud-assigned uuid 789D1360-DF3E-5826-9A36-45B7EB6E1C36), on IP address 127.64.103.51...
 status: Trying to establish network connectivity to vm_59 (cloud-assigned uuid 5C3389DD-2929-5499-BE9E-E1D50F5ED9B7), on IP address 118.3.19.218...
 status: Bypassing the sending of a copy of the code tree to vm_58 (127.64.103.51)...
 status: Bypassing the sending of a copy of the code tree to vm_59 (118.3.19.218)...
 status: Bypassing the sending of a copy of the code tree to vm_60 (198.69.239.253)...
 status: Bypassing generic VM post_boot configuration.
 status: Bypassing application-specific "setup" operations
AI object E5875E74-EFDB-546D-80B6-D5BE50F7D53B (named "ai_11") sucessfully attached to this experiment. It is ssh-accessible at the IP address 127.64.103.51 (789D1360-DF3E-5826-9A36-45B7EB6E1C36.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |141.98.169.121  |1358912185     |none                                    |CECA49F8-7F0A-5A2A-88AB-6FBD3C13A188
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |20.62.246.201   |1358912222     |none                                    |AA2AF411-5127-5D04-935C-861E9991D9CD
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |88.182.228.241  |1358912240     |none                                    |22D578AD-237A-5FD0-BF2E-8526B27365A8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |120.99.243.188  |1358912324     |none                                    |DA622B54-93C1-584C-A13C-A7741CB76ADD
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |88.118.248.29   |1358912327     |none                                    |DA3C582D-82E4-5D44-87E8-6543B22AB272
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |112.27.118.248  |1358912330     |none                                    |CEE9035A-0ECD-5ACA-AC70-99DCCEA9A76C
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |127.64.103.51   |1358912333     |none                                    |E5875E74-EFDB-546D-80B6-D5BE50F7D53B

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |svm_stub_vmc     |uuid
|vm_33         |client_daytrader    |nano32      |141.98.169.121  |simhostd1                  |LG                  |ai_3    |none       |none             |B3C70489-FAA9-51AC-93D3-4CD8824F4C4E
|vm_35         |db2                 |copper32    |6.33.25.203     |simhostc2                  |SUT                 |ai_3    |none       |none             |7EA11E39-05B7-55AC-9DCF-859A3ED6B774
|vm_34         |was                 |iron32      |122.233.44.250  |simhostb1                  |SUT                 |ai_3    |none       |none             |C5A52973-885F-5E10-BB14-312B7C7A42EC
|vm_36         |client_daytrader    |nano32      |20.62.246.201   |simhostd0                  |LG                  |ai_4    |none       |none             |5B0F2AA5-F9AD-59EA-BC73-1158F3CE045A
|vm_38         |db2                 |copper32    |94.33.53.77     |simhostc0                  |SUT                 |ai_4    |none       |none             |1D4ACB1F-496B-5AD7-AC7B-F48747946C30
|vm_37         |was                 |iron32      |209.154.30.29   |simhosta0                  |SUT                 |ai_4    |none       |none             |61D04C3E-05A5-58A9-90A7-0828A592F247
|vm_39         |netclient           |platinum64  |88.182.228.241  |simhosta1                  |SUT                 |ai_5    |none       |none             |20413B55-E05D-5FE7-BE95-75BB5E3B3EAA
|vm_40         |netserver           |iron32      |179.252.55.59   |simhostb4                  |SUT                 |ai_5    |none       |none             |E2836789-BFD8-56CC-89C2-515EDC558C26
|vm_48         |lb                  |iron32      |10.177.191.53   |simhostb2                  |SUT                 |ai_8    |none       |none             |83D03AFF-D1C0-5B18-84FA-8AC0F0E4E141
|vm_47         |client_daytrader    |nano32      |120.99.243.188  |simhostd3                  |LG                  |ai_8    |none       |none             |4ABA46E4-AA10-5094-BBA9-0F2D7DF457B0
|vm_50         |was                 |iron32      |239.137.217.196 |simhostc1                  |SUT                 |ai_8    |none       |none             |A8F030D6-28EE-598F-920B-BFB308B2FFA4
|vm_49         |was                 |iron32      |38.98.80.56     |simhostc0                  |SUT                 |ai_8    |none       |none             |AB15F359-12EB-529E-ABC2-5A1D520E4AA5
|vm_51         |db2                 |gold32      |193.150.194.155 |simhosta3                  |SUT                 |ai_8    |none       |none             |9F49A85B-F382-56DD-8C4A-A9FF15894397
|vm_52         |client_daytrader    |nano32      |88.118.248.29   |simhostd3                  |LG                  |ai_9    |none       |none             |68465B66-D241-5522-878F-F9C38DFDFBF2
|vm_53         |was                 |iron32      |79.105.170.135  |simhostc0                  |SUT                 |ai_9    |none       |none             |7DDD2C72-498A-5216-9AFF-F46656E5DB8C
|vm_54         |db2                 |copper32    |105.2.132.123   |simhostc4                  |SUT                 |ai_9    |none       |none             |870B0B43-85AC-5DEC-A725-5F057A1DF9F6
|vm_55         |client_daytrader    |nano32      |112.27.118.248  |simhostd3                  |LG                  |ai_10   |none       |none             |B3147FAC-13F6-57E8-B1E3-8D6C2C9F6C71
|vm_56         |was                 |iron32      |24.149.93.124   |simhostc0                  |SUT                 |ai_10   |none       |none             |457C0312-7579-52BF-A901-7349AE435981
|vm_57         |db2                 |copper32    |44.239.140.109  |simhostb3                  |SUT                 |ai_10   |none       |none             |B563AF56-FD90-5545-AEE4-A1E42A62E40A
|vm_58         |client_daytrader    |nano32      |127.64.103.51   |simhostd3                  |LG                  |ai_11   |none       |none             |6EC08E6D-952F-5A24-97AB-11CC49E55BFB
|vm_59         |was                 |iron32      |118.3.19.218    |simhostc1                  |SUT                 |ai_11   |none       |none             |BD51BD8C-EDFC-5847-8357-85D22499CD94
|vm_60         |db2                 |copper32    |198.69.239.253  |simhostc1                  |SUT                 |ai_11   |none       |none             |F6FD7741-9047-5CDF-870F-AAE9508CAEB4

The AI object ai_11, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |False                                
|sut                                     |1_x_client_daytrader->1_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.42M                         
|Redis Version                                        |2.6.7                         
|Uptime (in seconds)                                  |459650                        
|Total Connections Received                           |2820                          
|Total Commands Processed                             |4308209                       
|Number of Keys                                       |616                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.2.2                         
|Storage Size                                         |950272                        
|Data Size                                            |226800                        
|Index Size                                           |294336                        
|Average Object Size                                  |441.245136187                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |22                            
|SVMs                                                 |0                             
|AIs                                                  |7                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |22                            
|VMs ARRIVED                                          |57                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |7                             
|AIs ARRIVED                                          |11                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |198                           

 ---------------- VMC-WIDE COUNTERS ----------------
|66C80F72-DB30-5A63-B169-31C6A86D021F (simzone_c) VM RESERVATIONS|9                             
|D3E36C93-1427-5D86-BC91-CB0FA4A8E918 (simzone_d) VM RESERVATIONS|6                             
|FEBD66A7-E012-534C-A431-00CE123E50EF (simzone_a) VM RESERVATIONS|3                             
|9859398A-822B-50DC-8C37-7E3C5CBAE602 (simzone_b) VM RESERVATIONS|4                             

