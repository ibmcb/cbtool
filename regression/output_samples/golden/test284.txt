 status: Waiting for vm_58 (cloud-assigned uuid C2078BA1-87AF-5D3E-A238-D346660A43B3) to start...
 status: Waiting for vm_60 (cloud-assigned uuid 026CAD92-8534-51C0-85AD-683B2512ABD3) to start...
 status: Waiting for vm_59 (cloud-assigned uuid E414B69F-1923-52AD-AAE9-3A35E51E558C) to start...
 status: Trying to establish network connectivity to vm_58 (cloud-assigned uuid C2078BA1-87AF-5D3E-A238-D346660A43B3), on IP address 161.161.204.45...
 status: Trying to establish network connectivity to vm_59 (cloud-assigned uuid E414B69F-1923-52AD-AAE9-3A35E51E558C), on IP address 55.145.125.39...
 status: Trying to establish network connectivity to vm_60 (cloud-assigned uuid 026CAD92-8534-51C0-85AD-683B2512ABD3), on IP address 61.250.51.221...
 status: Bypassing the sending of a copy of the code tree to vm_58 (161.161.204.45)...
 status: Bypassing the sending of a copy of the code tree to vm_59 (55.145.125.39)...
 status: Bypassing the sending of a copy of the code tree to vm_60 (61.250.51.221)...
 status: Bypassing generic VM post_boot configuration on all VMs beloging to ai_11...
 status: Bypassing application-specific configuration on all VMs beloging to ai_11...
AI object AEB32216-5B53-507F-B4E3-1D41B06FB537 (named "ai_11") sucessfully attached to this experiment. It is ssh-accessible at the IP address 61.250.51.221 (026CAD92-8534-51C0-85AD-683B2512ABD3.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |145.30.115.73   |1372701642     |none                                    |0DDAA48E-8C7B-516D-9320-C823FB84989D
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |183.31.82.55    |1372701680     |none                                    |88BD1BD4-8E44-581E-8858-F7B071ADA048
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |130.243.129.99  |1372701699     |none                                    |22A8D4B3-C2D4-5433-8059-852E6D11B7EA
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |16.223.112.45   |1372701783     |none                                    |978B6948-3406-5ADD-A6C3-B5B4D876350E
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |86.79.103.219   |1372701787     |none                                    |38CA9A87-3CCF-52ED-B3FE-299A097F9CE0
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |153.119.113.224 |1372701790     |none                                    |865401ED-12CD-5203-9B13-532498EA05D1
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |61.250.51.221   |1372701794     |none                                    |AEB32216-5B53-507F-B4E3-1D41B06FB537

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_33         |client_daytrader    |nano32      |145.30.115.73   |simhostd0                  |LG                  |ai_3    |none       |E4B6738A-D2C7-5339-837B-A8448A06F1B1
|vm_35         |db2                 |copper32    |7.58.141.46     |simhostc4                  |SUT                 |ai_3    |none       |64C4C545-0B6F-565E-9511-3DAAD7E8A2B4
|vm_34         |was                 |iron32      |128.67.222.33   |simhosta2                  |SUT                 |ai_3    |none       |228A2675-52B3-556C-A927-93CAC163C469
|vm_37         |was                 |iron32      |184.196.201.117 |simhosta2                  |SUT                 |ai_4    |none       |6A633C4E-7999-5F41-941E-37B19F48870F
|vm_36         |client_daytrader    |nano32      |183.31.82.55    |simhostd0                  |LG                  |ai_4    |none       |3221F138-2036-5E98-85A6-67541540AD7F
|vm_38         |db2                 |copper32    |153.69.233.76   |simhostb0                  |SUT                 |ai_4    |none       |40FC0736-DF58-5EB7-B37A-D40614D8DCB1
|vm_40         |netclient           |platinum64  |130.243.129.99  |simhostb0                  |SUT                 |ai_5    |none       |857DE764-E6DB-54F8-837F-C8F0F5199F2B
|vm_39         |netserver           |iron32      |145.219.143.248 |simhostb3                  |SUT                 |ai_5    |none       |F9709FFF-07C5-53A1-90E6-C163F31DAA1B
|vm_49         |was                 |iron32      |60.249.142.82   |simhostb4                  |SUT                 |ai_8    |none       |7B0392CB-7A4C-53CC-89FE-660089C47FBA
|vm_47         |client_daytrader    |nano32      |16.223.112.45   |simhostd2                  |LG                  |ai_8    |none       |17018243-F1E9-5E54-BB2F-5AF3D77762EB
|vm_48         |lb                  |iron32      |163.59.94.10    |simhosta2                  |SUT                 |ai_8    |none       |8A2BB9BC-773B-5304-A63B-DF0E700640E9
|vm_50         |was                 |iron32      |105.26.247.147  |simhostb4                  |SUT                 |ai_8    |none       |9952EEE4-B9C7-5757-8FE9-9A0F8E5F4B53
|vm_51         |db2                 |gold32      |230.207.57.114  |simhosta1                  |SUT                 |ai_8    |none       |F87238D0-0FF4-5C7E-AA19-E61E33E80900
|vm_52         |client_daytrader    |nano32      |86.79.103.219   |simhostd2                  |LG                  |ai_9    |none       |29E597E8-672D-5B0F-8994-4E4AAC0FCD7B
|vm_53         |was                 |iron32      |187.191.186.86  |simhostb1                  |SUT                 |ai_9    |none       |CBA31D85-6164-5031-BF48-4864197DF4DD
|vm_54         |db2                 |copper32    |131.62.179.149  |simhostb2                  |SUT                 |ai_9    |none       |0E97D9D4-9D5D-5E36-AA1A-4C6E46E14492
|vm_55         |client_daytrader    |nano32      |153.119.113.224 |simhostd3                  |LG                  |ai_10   |none       |DD553669-4485-5106-A3BC-C205F267D0D1
|vm_56         |was                 |iron32      |255.75.70.67    |simhostc1                  |SUT                 |ai_10   |none       |9F72676A-FA0A-5917-B5AE-1C1FA768F6DD
|vm_57         |db2                 |copper32    |60.106.77.76    |simhosta3                  |SUT                 |ai_10   |none       |FA1EFA91-71C5-5189-B173-3BCFB47A69AE
|vm_58         |was                 |iron32      |161.161.204.45  |simhostb2                  |SUT                 |ai_11   |none       |013C3C57-DCA1-55A6-AEA3-3689A2E94535
|vm_60         |client_daytrader    |nano32      |61.250.51.221   |simhostd0                  |LG                  |ai_11   |none       |8F955973-767A-51CA-9E5D-A2880B542A98
|vm_59         |db2                 |copper32    |55.145.125.39   |simhosta4                  |SUT                 |ai_11   |none       |8E3959DE-73E0-567C-A8CC-AC35F354A1C6

The AI object ai_11, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |False                                
|sut                                     |1_x_client_daytrader->1_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |41.21M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11740573                      
|Total Connections Received                           |6492724                       
|Total Commands Processed                             |92774245                      
|Number of Keys                                       |616                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13966491648.0                 
|Data Size                                            |12134083052.0                 
|Index Size                                           |502889408                     
|Average Object Size                                  |6345.75999657                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |22                            
|SVMs                                                 |0                             
|AIs                                                  |7                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |22                            
|VMs ARRIVED                                          |57                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |7                             
|AIs ARRIVED                                          |11                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |198                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CB27A579-597C-52AC-B148-644E81A7D93C (simzone_d) VM RESERVATIONS|6                             
|D7D3EB8C-8813-524D-9C2D-7FD7626D4BCC (simzone_b) VM RESERVATIONS|8                             
|97FF48D4-05C4-54D7-B1D8-828EED2A2B20 (simzone_a) VM RESERVATIONS|6                             
|A708E2EA-221A-57ED-BAE8-15077C3E391A (simzone_c) VM RESERVATIONS|2                             

